www/content/examples/click-to-load.md

```diff
@@ -48,31 +48,31 @@ results (which will contain a button to load the *next* page of results).  And s
         }
       }
     }()
-    
+
     // routes
     init("/demo", function(request, params){
         var contacts = dataStore.contactsForPage(1)
         return tableTemplate(contacts)
     });
-    
+
     onGet(/\/contacts.*/, function(request, params){
         var page = parseInt(params['page']);
         var contacts = dataStore.contactsForPage(page)
         return rowsTemplate(page, contacts);
     });
-    
+
     // templates
     function tableTemplate(contacts) {
         return `<table><thead><tr><th>Name</th><th>Email</th><th>ID</th></tr></thead><tbody>
                 ${rowsTemplate(1, contacts)}
                 </tbody></table>`
     }
-    
+
     function rowsTemplate(page, contacts) {
       var txt = "";
       for (var i = 0; i < contacts.length; i++) {
         var c = contacts[i];
-        txt += "<tr><td>" + c.name + "</td><td>" + c.email + "</td><td>" + c.id + "</td></tr>\n";
+        txt += `<tr ${i==0&&'autofocus'}><td>  ${c.name}  </td><td>  ${c.email}  </td><td>  ${c.id}  </td></tr>\n`;
```

Comment:
Elements that are focusable should have a name so that AT has something to announce. Maybe move the autofocus to a tbody (having one tbody per page) and adding a label of some sort?

Reply:
removed

Reply:
Possible impl:
```js
txt += `<tbody autofocus aria-label="Page ${page}">`
for (var i = 0; i < contacts.length; i++) {
  txt += "<tr><td>" + c.name + "</td><td>" + c.email + "</td><td>" + c.id + "</td></tr>\n";
  txt += `<tr><td>  ${c.name}  </td><td>  ${c.email}  </td><td>  ${c.id}  </td></tr>\n`;
}
txt += `</tbody>`
```